app/main/views.py

```diff
@@ -87,25 +85,26 @@ def update_service(service_id):
     db.session.add(service)
     db.session.commit()
 
-    return jsonify(services=jsonify_service(service)), 200
+    return jsonify(services=jsonify_service(service)), http_status
```

Comment:
If we don't want to send back loads of data for each `PUT` request we could just return a `204 No content` and similarly not return a response for `201 Created`.
see: http://www.w3.org/Protocols/rfc2616/rfc2616-sec9.html#sec9.6